scenes/ui/options_menu/input_remapping.gd

```diff
@@ -0,0 +1,121 @@
+extends Control
+class_name InputRemappingUI
+##Recommended for use in a VBoxContainer
+##Fills itself with buttons to remap actions
+
+
+const BUTTON_ACTIVE_MODULATE:Color = Color.WHITE * 1.5
+
+#This needs tweaking
+@export var actionsAllowed:Array[StringName] = [
+#	&"j_left_click",
+#	&"j_right_click", 
+	&"j_toggle_bag",
+	&"j_toggle_equipment",
+	]
+@export var maxEventsPerAction:int = 1
+
+var currentAction:StringName
+var currentEvent:InputEvent
+var currentButton:Button
+
+func _ready() -> void:
+	update_list()
+	
+func update_list():
+	#Clear all children
+	for child in get_children(): child.queue_free
```

Comment:
I think you forgot the "()" for queue_free().
The child.queue_free() should also be placed on the next line